docs/serving/using-knative-services.md

```diff
@@ -0,0 +1,232 @@
+---
+title: "Using Knative Services"
+weight: 01
+linkTitle: "Using Knative Services"
+type: "docs"
+---
+
+Knative services are used to deploy an application. Knative Services attempt to model a more logical definition of a Service or Application. To achieve this, Services use the concept of Revisions which are each backed by a deployment with 2 Kubernetes Services.   
+
+For more information about Kubernetes Services, see the [Kubernetes documentation](https://kubernetes.io/docs/concepts/services-networking/service/).   
+
+Each Knative Service is defined by a Route and a Configuration, which have the same name as the Service, contained in a YAML file.
+
+## Before you begin
+
+To use Knative Services, you will need:
+* A Kubernetes cluster with [Knative installed](https://knative.dev/docs/install/index.html).
+
+## Creating a Knative Service
+To create an application, you need to create a YAML file that defines a Knative Service.
+This YAML file specifies metadata about the application, points to the hosted image of the app and allows the Service to be configured.
+
+This guide uses the [Hello World sample app in Go](https://knative.dev/docs/serving/samples/hello-world/helloworld-go) to demonstrate the structure of a Service YAML file and the basic workflow for deploying an app. These steps can be adapted for your own application if you have an image of it available on Docker Hub, Google Container Registry, or another container image registry.
+
+The Hello World sample app does the following:
+1. Reads an environment variable, `TARGET`, from the configuration .yaml file
+1. Prints `Hello World: \${TARGET}!`. If `TARGET` is not defined, it will print `NOT SPECIFIED`.
+
+### Procedure
+
+1. Create a new file named `service.yaml` containing the following information.
+
+```yml
+apiVersion: serving.knative.dev/v1 # (1)
+kind: Service
+metadata:
+ name: helloworld-go # (2)
+ namespace: default # (3)
+spec:
+ template:
+  spec:
+   containers:
+    - image: gcr.io/knative-samples/helloworld-go # (4)
+     env:
+      - name: TARGET # (5)
+       value: "Go Sample v1"
+```
+**(1)** Current version of Knative   
```

Comment:
For long term maintenance, can we avoid qualifying the lines of code and just use a list of some sort? [Example here](https://knative.dev/development/eventing/accessing-traces/#configuration-options) (a definition list would work great too).
That way if the YAML changes (ie new or removed lines) then we can avoid updating numbers in the codes and the list. [I like to avoid and reserve numbered lists for only items that require some type of linear order (due to the limits of markdown, the numbers below end up looking like lists (rather than key value mappings)].